### PR TITLE
feat(client): send logs to syslog

### LINKED ISSUE #2255: [question] linux client logs to syslog
is it possible to redirect linux client  logs to syslog istead of `/var/log/netbird/*`?

### PR DESCRIPTION
**Pull request by mrl5**
closes #2255

### Checklist
- [ ] Is it a bug fix
- [ ] Is a typo/documentation fix
- [x] Is a feature enhancement
- [ ] It is a refactor
- [ ] Created tests that fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

**Comment by mrl5** [comment]
hey, go is not really "my thing" yet but I think this changes should do the trick. any suggestions from reviewers more than welcome

File: util/log.go [hunk 1]
```diff
@@ -2,10 +2,13 @@ package util
 
 import (
 	"io"
+	"log/syslog"
 	"os"
 	"path/filepath"
+	"slices"
 
 	log "github.com/sirupsen/logrus"
+	lSyslog "github.com/sirupsen/logrus/hooks/syslog"
```
Comment (mlsmaycon) [inline util/log.go]: The syslog package is not available for Windows. We need to split the code that uses it into:

syslog_nonwindows.go <- with build tags like: `// +build !windows` and with a function that initializes the log with the syslog hook
syslog_windows.go <- with a function that does nothing

**Review by mlsmaycon — CHANGES_REQUESTED** [review]
@mrl5 thanks for your PR. I left a request to handle windows build. After fixing that you can mark the PR as ready.

**Review by mlsmaycon — APPROVED** [review]
thanks for your PR @mrl5

**Comment by mrl5** [comment]
thanks for merging @mlsmaycon ! here is a follow-up PR that fixes some sonar qube issues https://github.com/netbirdio/netbird/pull/2276

**Pull request merged**